vmware-event-router/cmd/router/main.go

```diff
@@ -63,6 +63,29 @@ func main() {
 	}
 	flag.Parse()
 
+	var lvl zapcore.Level
+	err := lvl.Set(logLevel)
+	if err != nil {
+		panic(err.Error())
+	}
+
+	// configure logger using defaults from the zap dev config
+	var zapCfg zap.Config
+	if logJSON {
```

Comment:
Does the difference between zap.NewProductionConfig() and zap.NewDevelopmentConfig() only in the format of the log aka. json vs non json? Also should both configurations include zapCfg.EncoderConfig.EncodeTime and zapCfg.EncoderConfig.EncodeLevel? Asking since the prod/dev naming below which is kind of ambiguous and we switch it with the json flag.

Reply:
Yeah, that's a little bit unfortunate (my choice/shortcut here). `NewDevelopmentConfig()` provides a good setup for our current VEBA users (smooth transition), whereas the defaults for `NewProductionConfig()` are much tighter and required some adjustments, e.g. time encoding. Alternatively, I could use the development config, diff it with production and only change the necessary fields for proper JSON encoding.

Reply:
I think this is not necessary for now (building the same config depending on the flag) and thinks look good as they are now. I think the simpler the better. Otherwise thanks for elaborating.

Reply:
Changed the config, can you please review and approve?